src/server.h

```diff
@@ -133,6 +133,9 @@ struct hdr_histogram;
 #define CLIENT_MEM_USAGE_BUCKET_MAX_LOG 33 /* Bucket for largest clients: sizes above 4GB (2^32) */
 #define CLIENT_MEM_USAGE_BUCKETS (1+CLIENT_MEM_USAGE_BUCKET_MAX_LOG-CLIENT_MEM_USAGE_BUCKET_MIN_LOG)
 
+/* static array len for each client */
+#define CLINET_STATIC_ARGV_LEN 128
```

Comment:
please lower this value, i think it's too high now, what about 16?

Reply:
I agree.

Reply:
@sundb @judeng  lowered it already.